src/libraries/System.Formats.Cbor/tests/Writer/CborWriterTests.cs

```diff
@@ -332,7 +333,36 @@ public static void WriteEncodedValue_ValidPayloadWithTrailingBytes_ShouldThrowAr
         [InlineData((CborConformanceMode)(-1))]
         public static void InvalidConformanceMode_ShouldThrowArgumentOutOfRangeException(CborConformanceMode mode)
         {
-            Assert.Throws<ArgumentOutOfRangeException>(() => new CborWriter(conformanceMode: mode));
+            Assert.Throws<ArgumentOutOfRangeException>("conformanceMode", () => new CborWriter(conformanceMode: mode));
+        }
+
+        [Theory]
+        [InlineData(-2)]
+        [InlineData(int.MinValue)]
+        public static void InvalidInitialCapacity_ShouldThrowArgumentOutOfRangeException(int capacity)
```

Comment:
Do we want to test for OOM in cases where `initialCapacity` is too large?

Reply:
In my opinion, no, for two reasons. The first is that doesn't seem like a concern of `CborWriter`'s tests. I don't think the writer should be testing how array allocation works, or doesn't work, in a way that it unconcerned with.

Second: We.. had.. some tests like this in S.S.Cryptography and I think we ended up disabling a few of the tests because some platforms (tvOS?) would crash or behave erratically during an OOM.